info_and_tips.md

```diff
@@ -52,30 +52,33 @@ If you want to `ssh` then that's cool too B-)
 
 * Chaining mulitple Sphero commands with <b>promises</b> and delays:
 ```javascript
-orb.color("green").roll(50,180).delay(1000).then(() => {
-  return orb.color("red").roll(50,270).delay(2000)
-}).then(() => {
-  return orb.roll(100, 0)
-})
+async function instructions() {
+  await orb.color("green").roll(50,180).delay(1000);
```

Comment:
This wasn't working:

`TypeError: orb.color(...).roll is not a function`

But it's also part of the old snippet

Reply:
await orb.color("green")
    await orb.roll(x).delay(1000)

or just

  orb.color("green")
  await orb.roll(x).delay(1000)